storage/statedb/database.go

```diff
@@ -822,9 +829,22 @@ func (db *Database) Commit(node common.Hash, report bool) error {
 	db.gcnodes, db.gcsize, db.gctime = 0, 0, 0
 	db.flushnodes, db.flushsize, db.flushtime = 0, 0, 0
 
+	if db.settingMigrationDBRequired(blockNum) {
+		db.diskDB.SetStateTrieMigrationDB(blockNum)
+	}
 	return nil
 }
 
+func (db *Database) settingMigrationDBRequired(blockNum uint64) bool {
+	if blockNum == 0 {
+		return false
+	}
+	if blockNum >= db.dataArchivingBlockNum && !db.diskDB.InMigration() {
```

Comment:
What if we want to archive blockchain data periodically? This seems that it archives data one time.

Reply:
Yes, as the policy has not been determined, this PR is to test if [1] creating a NEW_DB while the node is running and [2] using both OLD_DB and NEW_DB does not make any problem.